code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -755,63 +676,82 @@ GLOBAL_VAR_INIT(dynamic_forced_storyteller, null)
 /datum/game_mode/dynamic/proc/check_blocking(list/blocking_list, list/rule_list)
 	if(blocking_list.len > 0)
 		for(var/blocking in blocking_list)
-			for(var/datum/executed in rule_list)
+			for(var/_executed in rule_list)
+				var/datum/executed = _executed
 				if(blocking == executed.type)
 					return TRUE
 	return FALSE
 
-/// Checks if client age is age or older.
-/datum/game_mode/dynamic/proc/check_age(client/C, age)
```

Comment:
you actually need this, not sure why it's being removed

Reply:
I tried re adding this but it did nothing.  I'm still getting errors.  Tried adding it in some other folder, gave even more errors.